### PR TITLE
hide/show divs more gracefully 

### PR DESCRIPTION
**Pull request by vctt94**
fixes #915  and #926 and #927

### CODE REVIEW

**Comment by alexlyp** [comment]
opacity needs to be faster when closing rows.  I can still see the text/inputs after the row has already been moved up

**Comment by alexlyp** [comment]
When i clicked on "Send funds to another account": Cannot read property 'destination' of undefined

**Comment by alexlyp** [comment]
When I clicked on "Send all.." Received Nan for numeric attribute

**Comment by alexlyp** [comment]
Also please see mentioned issue #927 as well.

File: app/components/views/TransactionsPage/SendTab/OutputRow.js [hunk 1]
```diff
@@ -46,11 +44,11 @@ const SendOutputRow = ({
           />
         </div>
         {index === 0 && !isSendAll ? (
-          <div className="send-address-wallet-icon" onClick={onAddOutput}></div>
+          <div className="send-address-wallet-icon" onClick={()=>onAddOutput()}></div>
         ) : (index === 0 && isSendAll) ? (
           <div className="send-address-icon-spacer"></div>
         ) : (index === (outputs.length - 1)) && !isSendAll ? (
-          <div className="send-address-delete-icon" onClick={getOnRemoveOutput(index)}></div>
+          <div className="send-address-delete-icon" onClick={index=>getOnRemoveOutput(index)}></div>
```
[the remark this answers is not in the document]
Comment (vctt94) [reply app/components/views/TransactionsPage/SendTab/OutputRow.js]: On the first one the change was not required, but for getOnRemoveOutput(index), is required or it does not work

**Comment by alexlyp** [comment]
can you rebase this and test off master?  I'm currently getting an Assertion failed when trying to construct a ticket. I believe you need to add `unitDivisor: sel.unitDivisor` to send connectors to fix.

**Comment by vctt94** [comment]
How do you get this Assertion failed?

**Comment by alexlyp** [comment]
It's a bug on master, but just select send to account and enter any number into amount field, or enter an address and enter an amount.

Basically you just need it to issue a ConstructTransactionRequest

**Comment by alexlyp** [comment]
https://github.com/decred/decrediton/pull/932

**Comment by alexlyp** [comment]
I am still seeing a strange issue when sending to accounts and send all, steps to reproduce: 
- select an account with > 0 DCR for from.
- click "send to another account"
- enter an amount to send
- select another account to send funds to (note what the initial account and the new account is)
- confirm transaction

From what I can tell it sends to the initial account selected instead of the current selection.  Possibly it's not updating the generated address from the new account?

**Comment by vctt94** [comment]
Yeah, you were right, I believe now it is really fixed

**Pull request merged**